src/file/model/baseFile.ts

```diff
@@ -0,0 +1,12 @@
+import * as mongoose from 'mongoose';
+import { Document } from 'mongoose';
+import { baseModelName } from "./config"
+
+interface IBaseFile {
+    fileModel: string;
+}
+
+const baseFileOptions = { timestamps: true, collection: 'File', discriminatorKey: 'fileModel' };
```

Comment:
The collection name is already in the config.ts and the `discriminatorKey` should also be there is a constant

Reply:
done